src/plugins/controls/public/control_group/embeddable/control_group_container.tsx

```diff
@@ -63,6 +68,63 @@ export class ControlGroupContainer extends Container<
     return Promise.resolve();
   };
 
+  public getCreateControlButton = (
```

Comment:
These functions `getCreateControlButton` and  `getToolbarButtons ` are really important as an external facing API used via an embeddable.

To make it more obvious that these two functions are _intended_ for external use, we should consider adding API documentation to them.

Reply:
Great point - I added some documentation to both of these functions to make it clearer! :)